RadixWallet/Features/FactorSourceAccess/FactorSourceAccess.swift

```diff
@@ -0,0 +1,64 @@
+// MARK: - FactorSourceAccess
+public struct FactorSourceAccess: Sendable, FeatureReducer {
```

Comment:
Not sure if this is worth having as a Feature, could be just a view.

Reply:
Agree. However, we did briefly discuss with @CyonAlexRDX about the possibility of moving all the code from `DerivePublicKeys` and `SignWithFactorSource` into this new `FactorSourceAccess`. Not 100% sure this would be a better idea, but if we go on with it then it would definitely be a Feature.